### ISSUE TITLE
openstack node: file mounting and munge service start not ordered on reboot

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#310]
**Test procedure**: 1.2.20
**Version:**  r_4

When the openstack node reboots, we need to make sure the order of /trinity mounting and starting the munge service is ordered, otherwise slurm won't work.

**Comment by themisa** [comment]
```
Redirecting to /bin/systemctl status  munge.service
munge.service - MUNGE authentication service
   Loaded: loaded (/usr/lib/systemd/system/munge.service; enabled)
  Drop-In: /etc/systemd/system/munge.service.d
           customexec.conf
   Active: failed (Result: exit-code) since Wed 2015-06-24 11:20:23 UTC; 3min 53s ago
     Docs: man:munged(8)
  Process: 716 ExecStart=/usr/sbin/munged --key-file /cluster/etc/munge/munge.key (code=exited, status=1/FAILURE)

Jun 24 11:20:23 login-b.novalocal systemd[1]: Starting MUNGE authentication service...
Jun 24 11:20:23 login-b.novalocal munged[716]: munged: Error: Failed to check keyfile "/cluster/etc/munge/munge.key": No such file or directory
Jun 24 11:20:23 login-b.novalocal systemd[1]: munge.service: control process exited, code=exited status=1
Jun 24 11:20:23 login-b.novalocal systemd[1]: Failed to start MUNGE authentication service.
Jun 24 11:20:23 login-b.novalocal systemd[1]: Unit munge.service entered failed state.
```

**Comment by abhimukh** [comment]
This happened because on reboot  munge tries to start before the remote file systems are mounted, and the munge key file is in the /cluster directory.

I have fixed this in ( r5 02f7ecdbe220b1ff66c35fed26564c6a990b1dbf ) by adding a systemd dropin for the munge service. 

In earlier branches, manual workaround

```
# systemctl restart munge
```

**Comment by themisa** [comment]
Munge started ok. 
[root@login-a ~]# service munge status
Redirecting to /bin/systemctl status  munge.service
munge.service - MUNGE authentication service
   Loaded: loaded (/usr/lib/systemd/system/munge.service; enabled)
  Drop-In: /etc/systemd/system/munge.service.d
           customexec.conf, remote-fs.conf
   Active: active (running) since Wed 2015-07-01 13:34:04 UTC; 3min 57s ago
     Docs: man:munged(8)
  Process: 799 ExecStart=/usr/sbin/munged --key-file /cluster/etc/munge/munge.key (code=exited, status=0/SUCCESS)
 Main PID: 818 (munged)
   CGroup: /system.slice/munge.service
           818 /usr/sbin/munged --key-file /cluster/etc/munge/munge.key
